pkg/specgen/generate/pod_create.go

```diff
@@ -59,6 +59,11 @@ func MakePod(p *entities.PodSpec, rt *libpod.Runtime) (*libpod.Pod, error) {
 		if err != nil {
 			return nil, err
 		}
+	} else {
+		err := rt.SavePod(pod)
```

Comment:
If you have to repush.  Please add a comment here.  I wouldn't be able to make the link to logs when reading that line.

In case the code gets changed in the future and tests breaks, the authors may have a hard time figuring out what went wrong (or need to dig deep into the git history).

Reply:
fixed